components/invitation_modal/result_table.tsx

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import {UserProfile} from 'mattermost-redux/types/users';
+import {imageURLForUser, getLongDisplayName} from 'utils/utils';
+import {isGuest} from 'mattermost-redux/utils/user_utils';
+
+import EmailIcon from 'components/widgets/icons/mail_icon';
+import AlertIcon from 'components/widgets/icons/alert_icon';
+import GuestBadge from 'components/widgets/badges/guest_badge';
+import BotBadge from 'components/widgets/badges/bot_badge';
+import Avatar from 'components/widgets/users/avatar';
+
+import './result_table.scss';
+type InviteNotSent = {
```

Comment:
not sure why linter does not catch blank line after import statements, maybe not mandatory but definetely looks more readable :)